src/aux-server/aux-web/shared/scene/decorators/DimensionPositionDecorator.ts

```diff
@@ -382,6 +383,37 @@ export class DimensionPositionDecorator extends AuxBot3DDecoratorBase {
                 }
             }
 
+            // Update bounding objects if trasnformer bot moves.
+            const transformer = getBotTransformer(calc, this.bot3D.bot);
+
+            if (transformer) {
+                const bots =
```

Comment:
@Blitzy Would it be possible to look for the transformer bot by crawling up the object hierarchy? This would avoid the overhead of searching through every bot in the portal, as well as it would help ensure that scenarios where the bot is displayed in the portal twice (portal form bots) would work with transformers.

Reply:
@KallynGowdy I've changed the lookup to the use the 3d hierarchy. I also included a check to support being parented to a portal form (`DimensionGroup3D`).